apps/fz_http/lib/fz_http_web/controllers/auth_controller.ex

```diff
@@ -85,21 +85,14 @@ defmodule FzHttpWeb.AuthController do
           |> redirect(to: ~p"/")
       end
     else
-      {:error, reason} ->
-        msg = "OpenIDConnect Error: #{reason}"
-        Logger.warn(msg)
+      # Error verifying state, claims or fetching tokens
+      {:error, error} ->
```

Comment:
You might also want to add an `{:error, {:invalid_jwt, reason}}` clause for invalid tokens